sample/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /opt/intel/sgxsdk/environment
+
+if [ $# -lt 2 ]; then
+    exit
+fi
+
+# Only the replica node 0 generates keys.yaml
+if [ "$1$2" = "run0" ]; then
+    keytool -o /data/keys.yaml generate -u lib/libusig.signed.so
+else
+    until [ -f /data/keys.yaml ]; do
+        sleep 1
+    done
+fi
+
+peer "$@"
```

Comment:
It is suggested use `exec` so that the process becomes the container’s PID 1. This allows it to receive any Unix signals sent to the container.